src/slack.js

```diff
@@ -378,3 +372,33 @@ async function handleCSVUpload(event, bot) {
     comment: `${event.text || 'Your monthly invoice from VacuumLabs:'}\n_link_`,
   }
 }
+
+async function showError(state, channel, msg, ts = null) {
+  if (ts) {
+    return await boltApp.client.chat.update({
+      channel, ts, as_user: true,
+      blocks: [
+        {
+          type: 'section',
+          text: {
+            type: 'mrkdwn',
+            text: `:exclamation: ${msg}`,
+          },
+        },
+      ],
+    })
+  } else {
+    return await boltApp.client.chat.postMessage({
+      channel, ts, as_user: true,
+      blocks: [
+        {
+          type: 'section',
+          text: {
+            type: 'mrkdwn',
+            text: `:exclamation: ${msg}`,
+          },
+        },
+      ],
+    })
+  }
+}
```

Comment:
to keep the old function structure that wasn't duplicating code, something like this is possible:
```js
async function showError(state, channel, msg, ts = null) {
  await boltApp.client.chat[ts ? 'update' : 'postMessage']({
    channel,
    ts,
    as_user: true,
    blocks: [
      {
        type: 'section',
        text: {
          type: 'mrkdwn',
          text: `:exclamation: ${msg}`,
        },
      },
    ],
  })
}
```
I also removed the `return` keyword because there doesn't seem to be a usage of `showError` using the response

Reply:
thanks, I actually googled this out, but I thought it is some jQuery-specific syntax feature